src/device_manager/model/Device.js

```diff
@@ -16,6 +16,10 @@ export default Backbone.Model.extend({
     // If empty the width will be used
     widthMedia: null,
 
+    // The condition to use for media queries, eg. 'max-width'
+    mediaCondition: null,
+    defaultMediaCondition: null,
```

Comment:
I think there is no need for defaultMediaCondition, you can get it directly from the config